### ISSUE TITLE
feature: Brand Apps, eg: Google / Amazon / Microsoft

### ISSUE THREAD

**Issue by marcusquinn** [webcatalog/webcatalog-legacy#986]
# Feature Request
**Specifications:**
- App Version:
- OS (macOS, Windows, Linux):

**Description:**
Similar to what you've done for Work / School / Life

It would be super-handy to have ready-made brand/category Apps like; Google, Amazon, Ebay, Freshworks, Microsoft, Chat, CMS, Email, Music, News, Research, Social, Video, Websites, Writing etc.

Where you create the App (similar to Work / School / Life) but then tick all the related Child apps you want in it.

Basically, the combination of Singlebox & Webcatalog advantages. The biggest advantage then being a less cluttered Dock/Taskbar where all the many Apps can created in one of these Singlebox groups very quickly, customised and updated quickly too.

**Comment by marcusquinn** [comment]
Only just tried the feature for "Don't specify a URL".

I actually think that almost covers this given you can search by brand name.

Maybe change that label to "App Group" or "Multi-site" or whatever you think describes it better. I don't think you need the tick-box label to say "Don't specify a URL" as the URL field is greyed-out when you tick it anyway.

**Comment by quanglam2807** [comment]
@marcusquinn This is actual great feedback! I'll see what I can do to make it clearer to users.

**Comment by marcusquinn** [comment]
Cool, that works well. It's a great feature!

**Comment by quanglam2807** [comment]
I still think the Brand Apps idea is great. But I’ll need to improve the catalog experience first before adding it. There’re too many apps already.

**Comment by marcusquinn** [comment]
> I still think the Brand Apps idea is great. But I’ll need to improve the catalog experience first before adding it. There’re too many apps already.

Cool, you may want to make them so they share cookies & sessions by default given they usually have a common SSO login.
